index.html

```diff
@@ -59,12 +61,12 @@ <h1 class="animation-h">I’m Victor Ozoede Glad to see you!</h1>
       </ul>  
     </section> 
     
-    <main class="grid">
+    <main class="grid main"> 
       <!-- First section starts -->
       <section class="work-one work">
         <img class ="img-work img-work-one" src="img/Portfolio1.svg" alt="Facebook 360">
           <div class="body-content">
-          <h2>Facebook 360</h2>
+          <h2 class="bold">Facebook 360</h2>
             <ul class="work-top">
               <li class="canopy">CANOPY</li>
```

Comment:
- [ ] As previously requested, for this milestone you are required to display all your projects dynamically using JavaScript. Not only do you have all the projects in the array of objects within the script.js file, but also, they exist in the index.html file. I took the time to set up your repo locally, and it turns out the projects displayed are only the ones in the index.html file
- Remove all the projects from the inde.html file and only depend on the array of objects from the script.js file to display all your projects.

Reply:
Good day , 
The highlighted part is not part of this milestone.
This milestone started from line 94 and ended at line 123.

Thank you.